new/LIT7325FourCorners.xml

```diff
@@ -0,0 +1,64 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7325FourCorners" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ጸሎት፡ በእንተ፡ መፍትሔ፡ ሥራይ፡ እለ፡ ይገብሩ፡ ሥራየ፡ ፬መዓዝነ፡ ዓለም፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Ṣalot baʾǝnta maftǝḥe śǝrāy ʾǝlla yǝgabbǝru śǝrāya 4-maʿāzǝna ʿālam</title>
+                <title xml:lang="en" corresp="#t1">Prayer for the loosening of witchcraft, witchcraft which is exercized (in the) four corners of the
```

Comment:
I think the translation should be "Prayer for the loosening of witchcraft. Those who perform witchcraft  (in the) four corners of the world …"